internal/cloudinfo/providers/amazon/cloudinfo.go

```diff
@@ -553,7 +553,7 @@ func (e *Ec2Infoer) GetServiceImages(service, region string) ([]types.Image, err
 	serviceImages := make([]types.Image, 0)
 	switch service {
 	case svcEks:
-		for _, k8sVersion := range []string{"1.16", "1.17", "1.18", "1.19", "1.20"} {
+		for _, k8sVersion := range []string{"1.16", "1.17", "1.18", "1.19", "1.20", "1.21"} {
```

Comment:
**Required**: we **MUST** also update the `GetVersions()` [returned version list](https://github.com/banzaicloud/cloudinfo/blob/7832ffa6f2c958c69a8ffa99dbee4fd4b3ada8fb/internal/cloudinfo/providers/amazon/cloudinfo.go#L620) as described in the EKS version addition guide.

Reply:
fixed